app/cells/datagrid_filter/datagrid_filter.yml

```diff
@@ -1,14 +1,9 @@
 description: "Renders filters for Datagrid. Filters are retrieved from a grid instance via it's `filters` method. See [the Datagrid docs](https://github.com/bogdan/datagrid/wiki/Filters) on how to implement filters for a grid."
 examples:
   - slim: |
-      - string_filter = OpenStruct.new options: {},
-                                      type: :string,
-                                      name: 'string',
-                                      header: 'String'
-      - select_filter = OpenStruct.new options: { select: -> { [['Baz', 'baz']] } },
+      - dummy_filter = OpenStruct.new options: { select: -> { [['Baz', 'baz']] } },
                                       type: :enum,
-                                      name: 'select',
-                                      header: 'Select'
+                                      name: 'bar',
+                                      header: 'Bar'
       - bootstrap_form_for(:foo, url: '/') do |f|
-        = ui_component('datagrid_filter', filter: string_filter, form: f)
-        = ui_component('datagrid_filter', filter: select_filter, form: f)
+        = ui_component('datagrid_filter', filter: dummy_filter, form: f)
```

Comment:
Is this intentional or a result of the rebase? I think `string_filter` and `select_filter` have been added just recently.